src/classes/PMT_PaymentCreator.cls

```diff
@@ -436,7 +458,22 @@ public class PMT_PaymentCreator {
         return (amount1Rounded >= amount2Rounded);
     }
 
-    /*******************************************************************************************************
+    /**
+    * @description specifies whether Payments should be excluded for the Opportunity
+    * @param opp the Opportunity to test.
+    * @return boolean true if Payments excluded for the Opportunity
+    */ 
+    private static boolean isNoPaymentForOpp(Opportunity opp) {
```

Comment:
- Let's specify in the Apex doc when a payment should not be created for an Opp.
- Data types should start with a capital letter, ie `Boolean`, `Id.
- `if` blocks should be enclosed in `{` and `}`
- I see this is an old method, so I am not going to ask for the rename :)